pytket/docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-sphinx >= 4.5, < 6.2.0
+sphinx >= 4.5
```

Comment:
Do we want to set a new restriction for example < 7 or so? Just to make sure we don't get surprised by api changes?

Reply:
Sure I can do that

Reply:
Done